src/components/EpisodePreview/EpisodePreview.jsx

```diff
@@ -0,0 +1,128 @@
+import React, { useContext } from "react";
+import {
+	EpisodesContext as Episodes
+} from "COMPONENTS/Episodes";
+
+export default function EpisodePreview(props){
+
+	// CONTEXT
+	//-----------------------------
+	const episodes = useContext(Episodes);
+
+
+	// RENDER
+	// ----------------------------
+	const { 
+		episodeId,
+		index,
+		episodeNo,
+		title,
+		thumbnail: {
+			src: thumbSrc,
+			description: thumbDesc
+		},
+		trailer: {
+			src: trailerSrc
+		}
+	} = props;
+
+	// handle visibility of active episode and episode preview overlay
+	const isActiveEpisode 	= episodes.get.activeIndex === index;
+	const thisOverlayActive = isActiveEpisode && episodes.get.overlayActive;
```

Comment:
`this` is a very loaded word, so we shouldn't use it in variable names otherwise it just looks like we're missing a `.` (ie. `this.overlayActive` or something) - also, you've set a precedent for naming in the line above with `isActiveEpisode` and so we might as well continue it with `isOverlayActive`.